.github/workflows/test.yml

```diff
@@ -31,11 +31,15 @@ jobs:
   pwa-kit:
     strategy:
       matrix:
-        node: [14, 16]
-        npm: [6, 7, 8]
+        node: [14, 16, 18]
+        npm: [6, 7, 8, 9]
         exclude:
           - node: 16
             npm: 6
+          - node: 18
+            npm: 6
+          - node: 14
```

Comment:
do you mean
```suggestion
          - node: 18
```

Reply:
No, note that they are inside of the matrix exclude section, which means that we want to exclude the builds with the combinations: Node 16 and NPM 6, Node 18 and NPM 6 and Node 14 and NPM 9.

We do that because out of the box:
Node 14 bundles NPM 6.
Node 16.0.x - 16.10.x bundles NPM 7.
Node 16.11.x - 16.19.x bundles NPM 8.
Node 18 bundles NPM 9.

When devs switch their Node version, using nvm or installing the Node bundle, they typically use the NPM version that comes bundled with the Node version.

We're removing build with the least likely combinations of Node and NPM to reduce the number of GA builds.